src/lib/updating-element.ts

```diff
@@ -415,6 +444,12 @@ export abstract class UpdatingElement extends HTMLElement {
    */
   protected initialize() {
     this.renderRoot = this.createRenderRoot();
+    // Note, if renderRoot is not a shadowRoot, styles would/could apply to the
+    // element's getRootNode(). While this could be done, we're choosing not to
+    // support this now since it would require different logic around de-duping.
+    if (this.renderRoot instanceof window.ShadowRoot) {
```

Comment:
Will fail in a env that does not supports shadow dom and not is pollyfied. Such env would be supported at all?

Reply:
Thanks, fixed.